test/unit_tests/apps/teachers-digital-platform/js/search-spec.js

```diff
@@ -247,7 +247,7 @@ describe( 'The TDP search page', () => {
     };
     global.XMLHttpRequest = jest.fn( () => mockXHR );
     const parentCheckbox = document.querySelector( '#topic--earn' );
-    const form = document.querySelector( 'form#filter-form' );
+    // const form = document.querySelector( 'form#filter-form' );
```

Comment:
Seems like it could be deleted?

Reply:
Yes, this is an unused variable so I deleted it.